src/dmd/astbase.d

```diff
@@ -3227,35 +3238,43 @@ struct ASTBase
                     break;
 
                 case MODFlags.const_:
-                    cto = t;
+                    getMcache();
+                    mcache.cto = t;
```

Comment:
y u no `getMcache().cto = t;` as you did below?

Reply:
eh, potayto potahto